### PR TITLE
Update CHANGELOG DKIM

### PR DESCRIPTION
**Pull request by remk**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -11,6 +11,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 ### Changed
 - Migration to ElasticSearch 6.3
 - Blob export to LinShare
+- Native DKIM support for outgoing mails. The mailets from james-jdkim have been included in main James project.
```
Comment (chibenwa) [inline CHANGELOG.md]: This is specific for Guice but did you do it for Spring?

**Comment by chibenwa** [comment]
test this please

**Comment by trantienduchn** [comment]
read it

**Comment by chibenwa** [comment]
test this please

**Comment by chibenwa** [comment]
test this please

File: CHANGELOG.md (hunk 1 above)
Comment (mbaechler) [reply CHANGELOG.md]: signing doesn't require injection

Comment (chibenwa) [reply CHANGELOG.md]:
```
benwa@horizon ../apache/onssd/james-project/server/app (git)-[master] % mvn dependency:tree | grep dkim                              
mvn dependency:tree  14,23s user 0,50s system 396% cpu 3,716 total
grep --color=auto dkim  0,02s user 0,02s system 1% cpu 3,716 total
```

I suppose spring users can not use it :-(

Comment (mbaechler) [reply CHANGELOG.md]: @chibenwa : I don't understand the output of your grep.

Comment (chibenwa) [reply CHANGELOG.md]: server/app does not depend on james-server-mailet-dkim which means the mailet is not usable currently on top of Spring. Either you add the dependency - and test it - or you recognise that this is a guice only feature.

Comment (mbaechler) [reply CHANGELOG.md]: Ok, good point, spring doesn't have the dependency.
However, requiring to test it is stricter than our current policy. By now, if smoke tests work, we are happy enough, right?
I'll add the dependency to this PR.

**Comment by mbaechler** [comment]
I can't push force here, I opened a new PR with the dependency update : https://github.com/linagora/james-project/pull/2657

**Comment by chibenwa** [comment]
test this please

(build result lost)

**Comment by trantienduchn** [comment]
```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.7.0:compile (default-compile) on project metrics-api: Fatal error compiling: invalid flag: --release -> [Help 1]
[e450ae00ec06473b0cc29866b293c259b698bdd5] Progress (4): 3.7/4.4 MB | 3.3/13 MB | 2.5/4.6 MB | 1.2/3.3 MB
P[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.7.0:compile (default-compile) on project javax-mail-extension: Fatal error compiling: invalid flag: --release -> [Help 1]
```
weird, test this please

**Comment by rouazana** [comment]
replaced by https://github.com/linagora/james-project/pull/2657

**Pull request closed without merging**